showcase/tsconfig.json

```diff
@@ -12,7 +12,9 @@
     ],
     "paths": {
       "@angular/*": ["../node_modules/@angular/*"],
-      "@nationalbankbelgium/*": ["../node_modules/@nationalbankbelgium/*"]
+      "@nationalbankbelgium/*": ["../node_modules/@nationalbankbelgium/*"],
+      "core-js/es7/reflect": ["../node_modules/core-js/proposals/reflect-metadata"],
+      "core-js/es6/*": ["../node_modules/core-js/es/*"]
```

Comment:
are these needed? because it seems you removed them from `starter/src/polyfills.browser.ts`

Reply:
In fact this is the workaround to get Angular CLI working with the latest `core-js`... See https://github.com/angular/angular-cli/issues/13954... I would like to add a FIXME somewhere to mention this but where do I add it? This is a json file and we cannot put comments in here :(

Reply:
This is also related to the issue you mentioned in the other remarks: angular/angular#29531

Reply:
Ok, I understand the fix now. 😄
As for the .json comments. You are right we shouldn't/can't comment there. Maybe we should start a *known issues* section in our README.md I actually think this would also provide better visibility to active issues (for us) then the FIXME comments we add in code. 🤔 
@SuperITMan @christophercr what do you think?

Reply:
Thinking about it again **known issues** might be the wrong title. 🤔

Reply:
I think it could be a good idea 😊 
I'll have a look asap

Reply:
I added a comment about this workaround in the `POLYFILLS.md` and also in `polyfills.browser.ts`

Reply:
Probably best. 😄